2-mandatory/3-financial-times.js

```diff
@@ -23,23 +32,36 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+    let arrFinal = [];
+    for (let i = 0 ; i < allArticleTitles.length ; i++){
+       for (let j = 0 ; j < allArticleTitles[i].length ; j++){
+            if (allArticleTitles[i][j] >= "0" && allArticleTitles[i][j] <= "9") {
```

Comment:
well done! Can you think about how you could refactor this so that you create a new function that checks the letters of each title, to avoid having two nested `for` loops?